test/unit/reductions/moments/test_moments_demographic_parity.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+
+from fairlearn.reductions import DemographicParity
+from fairlearn.reductions._moments.moment import _EVENT, _GROUP_ID, _SIGN
+
+from .data_generator import simple_binary_threshold_data
+
+
+def test_construct_and_load():
+    dp = DemographicParity()
+    assert dp.short_name == "DemographicParity"
+
+    num_samples_a0 = 10
```

Comment:
so "a" is sensitive feature?
I can only assume this confuses other people like it confused me. At least add a comment that that's what it means, or let's just name it num_samples_sensitive_feature0

Reply:
Documentation added